cmd/clusterctl/examples/aws/generate-yaml.sh

```diff
@@ -104,16 +104,16 @@ echo "Done copying ${ADDONS_FILE}"
 CREDENTIALS="$(${CLUSTERAWSADM} alpha bootstrap encode-aws-credentials)"
 echo "Generated credentials"
 
-PROVIDER_COMPONENTS="$(cat ${PROVIDER_COMPONENTS_SRC})"
-
-echo -e "${PROVIDER_COMPONENTS}\n${CREDENTIALS}" > "${PROVIDER_COMPONENTS_FILE}"
-echo "Done writing ${PROVIDER_COMPONENTS_FILE}"
-echo "WARNING: ${PROVIDER_COMPONENTS_FILE} includes credentials"
-
 if [ -f $PROVIDER_COMPONENTS_SRC_DEV ]; then
   PROVIDER_COMPONENTS_DEV=$(cat ${PROVIDER_COMPONENTS_SRC_DEV})
 
   echo -e "${PROVIDER_COMPONENTS_DEV}\n${CREDENTIALS}" > "${PROVIDER_COMPONENTS_FILE_DEV}"
   echo "Done writing ${PROVIDER_COMPONENTS_FILE_DEV}"
   echo "WARNING: ${PROVIDER_COMPONENTS_FILE_DEV} includes credentials"
+else
```

Comment:
I don't super love this bash change because of our current manual release process. I agree with it being technically correct, but it's more inconvenient and I can see this leading to a few minutes of confusion while someone works out they have to delete their dev file to do a release build.